system/login.go

```diff
@@ -0,0 +1,70 @@
+package system
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// GetCurrentLogins returns the user ids of the currently logged in users
+func GetCurrentLogins() []string {
```

Comment:
@angelabriel  For what exactly is this option needed?

Reply:
If the UserTasksMax value should be changed by 'saptune note apply <NoteID>' we create a drop in file (/etc/systemd/login.conf.d/saptune-DefaultUserTasksMax.conf), so that all users, which are log in later, will get the new setting. Additionally we use a systemctl command to change the value during runtime for all currently logged in users.  And for this command we need to know which users are currently logged in.
 systemctl  --runtime set-property user-<uid>.slice TasksMax=
So there is no need for a reboot any longer and that was the reason to decide to set UserTasksMax during apply/revert operation and NOT during the postinstall of the package installation.

Reply:
Hum... I see... Clever!